doc/source/cd/methods/classifierdrift.ipynb

```diff
@@ -45,9 +45,9 @@
     "\n",
     "* `preprocess_fn`: Function to preprocess the data before computing the data drift metrics.\n",
     "\n",
-    "* `preds_type`: Whether the model outputs 'probs' (probabilities) or 'logits'.\n",
+    "* `preds_type`: Whether the model outputs 'probs' (for 'tensorflow', 'pytorch', 'sklearn' models), 'logits' (for 'pytorch', 'tensorflow' models), 'scores' (for 'sklearn' models if `decision_function` is supported).\n",
```

Comment:
Please add `probabilities` back in the description to be explicit about what `probs` is.